packages/groq-builder/src/commands/conditional.ts

```diff
@@ -13,9 +13,9 @@ import { ProjectionMap } from "./projection-types";
 declare module "../groq-builder" {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   export interface GroqBuilder<TResult, TRootConfig> {
-    conditional$<
+    conditional<
```

Comment:
Do you know what the `$` signified before, or was it just there out of habit? :sweat_smile:

Reply:
Initially, there were a few methods that had "loose types". I wanted a way to distinguish those methods, so that it was easy to tell that you weren't fully type safe. 
For example,  these conditional expressions are just strings, so they're not safe. 
I used the `$` to mark these unsafe methods. 
However,  now that I've implemented nearly the full api, I've realized that the only "loose types" are these conditions. So ultimately,  I don't think it's worthwhile to use a naming convention to indicate this.  I'd rather use documentation.